dvc/repo/experiments/show.py

```diff
@@ -40,6 +40,16 @@ def _collect_experiment_commit(
         if params:
             res["params"] = params
 
+        res["deps"] = {
+            dep.def_path: {
+                "hash": dep.hash_info.value,
+                "size": dep.meta.size,
+                "nfiles": dep.meta.nfiles,
+            }
+            for dep in repo.index.deps
+            if type(dep).__name__ != "ParamsDependency" and dep.is_in_repo
```

Comment:
```suggestion
            if not isinstance(dep, ParamsDependency) and dep.is_in_repo
```

Reply:
@daavoo, do we really need `dep.is_in_repo`? It may be slow for "in-repo" dependencies. If we don't need it strictly, we could just do:
```suggestion
            if not isinstance(dep, (ParamsDependency, RepoDependency))
```